docs(Blocky): Add more to Setup-Guide (#12235)
truecharts/charts: charts/enterprise/blocky/docs/setup-guide.md

# Blocky Setup Guide

> This will guide you through the basic setup of Blocky which is the preferred DNS solution for TrueCharts. This guide will cover basic setup options which will get you up and running and is not all inclusive.  Setting up your systems to point to Blocky is out of scope of the document.

Reviewer: ```suggestion
This will guide you through the basic setup of Blocky which is the preferred DNS solution for TrueCharts. This guide will cover basic setup options which will get you up and running and is not all inclusive.  Configuring your devices to use Blocky is out of scope of this guide.
```

## Upstream DNS

Blocky has the following DNS entries configured by default.  They can be overridden to your personal preferences or left as default. Here are some common ones:

- 1.1.1.1
- 1.0.0.1
- 8.8.8.8
- 8.8.4.4
- 9.9.9.9
- 149.112.112.112
- 208.67.222.222
- 208.67.220.220
- 8.26.56.26
- 8.20.247.20
- 185.228.168.9
- 185.228.169.9
- 76.76.19.19
- 76.223.122.150
- 76.76.2.0
- 76.76.10.0

> Blocky supports 3 methods for upstream DNS.  You can choose multiple options, but I'm not sure if the order matters.

Reviewer: ```suggestion
Blocky supports 3 methods for upstream DNS.  You can use any combination of the below.
```

## Bootstrap DNS

If you entered a non-IP address (meaning you used a domain name) for DoT or DoH, then you need to ensure that a bootstrap DNS provider
is configured to resolve the DoT or DoH address. This provider can be any UDP upstream DNS.

> In the below example I am using Google DNS.  I don't know if you need a bootstrap DNS if you have a UDP server listed above.

Reviewer: ```suggestion
In the below example I am using Google DNS.
```

1. Pick a Group Name for your blocklists.
2. Add List entries for each blocklist by URL.
   ![blocky-blacklist](./img/blocky-blacklist.png)
3. Add a Clients Group Block and set Client Group Name to `default`
4. Under Groups Entry enter the Group name you used above.
   ![blocky-blacklist-group](./img/blocky-blacklist-group.png)

## Networking

If you want to use Blocky on your local network to take advantage of the filtering above, or the k8s-gateway below, you 

> need to ensure your loadbalancer IP address is set and won't change.

Reviewer: ```suggestion
need to setup a static ip address and leave the default setting DNS TCP Service and DNS UDP Service as LoadBalancer.
```

> You need to configure the systems on your network to use the loadbalancer IP address configured above.  This will either be done per system (seting a `static IP` for your Network Interface and having the apps use it)

Reviewer: ```suggestion
You need to configure the devices on your network to use the static IP address configured above.  This can be done per device or by changing the DHCP or DNS settings on your router.
```

> or by changing the DHCP settings on your router.  

Reviewer: ```suggestion
```
